core/types/derive_sha.go

```diff
@@ -17,25 +17,141 @@
 package types
 
 import (
-	"bytes"
-
 	"github.com/ledgerwatch/turbo-geth/common"
 	"github.com/ledgerwatch/turbo-geth/rlp"
 	"github.com/ledgerwatch/turbo-geth/trie"
 )
 
+var emptyHash = trie.New(common.Hash{}).Hash()
```

Comment:
Does it end up being the same as this one: https://github.com/ledgerwatch/turbo-geth/blob/28389a1cb40ffd2640f323c780538e5ca8de7810/trie/trie.go#L33 ?

Reply:
ah, it's a constant 😆
yeah, it is the same, and I'll fix the PR to actually use this constant.

Reply:
fixed